pyrex.ini

```diff
@@ -67,6 +67,7 @@ tempdir = ${build:builddir}/pyrex
 # A list of directories that should be bound when running in the Docker image
 %bind =
 %    ${build:oeroot}
+%    ${build:builddir}
```

Comment:
I suppose what this means is that previously, the builddir was implicitly mounted just because it happened to usually be a child of the main Yocto working copy?

Reply:
Correct. This was actually a bug in the default configuration that the CI caught. I was going to cherry-pick it back to master, but figured I was merging this anyway.

Reply:
Great, figured so. Thanks.